### ISSUE TITLE
Motion Detected! Failed to load attachment: Response code was unacceptable: 404. Version: 1.3.1

### ISSUE THREAD

**Issue by SleepyMoz** [valentinfrlch/ha-llmvision#137]
### Bug Description
I hav2 filled in the configuration but did not fill in any type or authentication which is probably why this bug occurs.

**Version**: 
Latest:Latest

![image](https://github.com/user-attachments/assets/7f7b1542-9bcb-4896-85f5-1035f0cb95e3)

**Comment by valentinfrlch** [comment]
Please some more information. What authentication did you omit? Is this the blueprint and you're using Frigate?

**Comment by SleepyMoz** [comment]
I have a dlink camera that I wired up with the generic camera module from HA. Within the plug-in I was not able to authenticate though after an investigation I do not think it is the root of the issue. I am using the blueprint, though I had to manually patch it as importing it into HA did not work for me. I edited the last line as HA was complaining with the error: while parsing a block mapping in "<unicode string>", line 1, column 1: blueprint: ^ expected <block end>, but found '-' in "<unicode string>", line 432, column 1: - delay: 00:{{cooldown|int}}:00 ^
